controllers/front/action.php

```diff
@@ -17,25 +18,366 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
+
+use PrestaShop\Module\BlockWishlist\WishList;
+use PrestaShop\PrestaShop\Core\Product\ProductListingPresenter;
+use PrestaShop\PrestaShop\Adapter\Image\ImageRetriever;
+use PrestaShop\PrestaShop\Adapter\Product\PriceFormatter;
+use PrestaShop\PrestaShop\Adapter\Product\ProductColorsRetriever;
+use PrestaShop\PrestaShop\Core\Product\ProductPresentationSettings;
+
+
 class BlockWishlistActionModuleFrontController extends ModuleFrontController
 {
     public function initContent()
     {
         parent::initContent();
 
+        if (!$this->context->customer->isLogged()) {
+            $this->ajaxDie(
```

Comment:
```suggestion
            $this->ajaxRender(
```
https://github.com/PrestaShop/PrestaShop/blob/1.7.6.x/classes/controller/Controller.php#L707